src/Business Foundation/App/NoSeriesCopilot/src/Register/NoSeriesCopilotInstall.Codeunit.al

```diff
@@ -13,10 +13,11 @@ codeunit 290 "No. Series Copilot Install"
     InherentPermissions = X;
```

Comment:
Quick comment for when this is re-enabled. This should be run on upgrade, not install. We have another install codeunit, but it's backed up by an upgrade and it's only because the first install of BusinessFoundation should be an install (even that is not always completely true). After that, this will be an upgrade.

Reply:
Got will. Will address when re-enabling. Thanks 🙏